library/src/main/java/com/novoda/downloadmanager/lib/DownloadInfo.java

```diff
@@ -193,53 +80,50 @@ private Long getLong(String column) {
      */
     public static final String EXTRA_IS_WIFI_REQUIRED = "isWifiRequired";
 
-    public long mId;
-    public String mUri;
-    /**
-     * Add to check if scannable i.e. we don't want internal files to be scanned
-     */
-    public boolean mScannable;
-    public boolean mNoIntegrity;
-    public String mHint;
-    public String mFileName;
-    public String mMimeType;
-    public int mDestination;
-    public int mVisibility;
-    public int mControl;
-    public int mStatus;
-    public int mNumFailed;
-    public int mRetryAfter;
-    public long mLastMod;
-    public String mClass;
-    public String mExtras;
-    public String mCookies;
-    public String mUserAgent;
-    public String mReferer;
-    public long mTotalBytes;
-    public long mCurrentBytes;
-    public String mETag;
-    public int mUid;
-    public int mMediaScanned;
-    public boolean mDeleted;
-    public String mMediaProviderUri;
-    public int mAllowedNetworkTypes;
-    public boolean mAllowRoaming;
-    public boolean mAllowMetered;
-    public int mBypassRecommendedSizeLimit;
-    public long batchId;
-
-    private List<Pair<String, String>> mRequestHeaders = new ArrayList<Pair<String, String>>();
+    private long id;
+    private String uri;
+    private boolean scannable;
+    private boolean noIntegrity;
+    private String hint;
+    private String fileName;
+    private String mimeType;
+    private int destination;
+    private int control;
+    private int status;
+    private int numFailed;
+    private int retryAfter;
+    private long lastMod;
+    private String notificationClassName;
+    private String extras;
+    private String cookies;
+    private String userAgent;
+    private String referer;
+    private long totalBytes;
+    private long currentBytes;
+    private String eTag;
+    private int uid;
+    private int mediaScanned;
+    private boolean deleted;
+    private String mediaProviderUri;
+    private int allowedNetworkTypes;
+    private boolean allowRoaming;
+    private boolean allowMetered;
+    private int bypassRecommendedSizeLimit;
+    private long batchId;
 
     /**
      * Result of last {DownloadThread} started by
+<<<<<<< HEAD
+     * {@link #isReadyToDownload(CollatedDownloadInfo)} && {@link #startDownloadIfNotActive(ExecutorService, StorageManager, DownloadNotifier)}.
+=======
      * {@link #isReadyToDownload(DownloadBatch)} && {@link #startDownloadIfNotActive(ExecutorService)}.
+>>>>>>> master
      */
-    private Future<?> mSubmittedTask;
+    private Future<?> submittedThread;
```

Comment:
I suppose yeah, I didn't think it was a problem here because it's `private` and it has no getter. I'll go with the consensus.